pyproject.toml

```diff
@@ -1,17 +1,17 @@
 [tool.black]
 line-length = 120
 preview = true
+extend-exclude = "SentinelHub/resources.py|SentinelHub/external|SentinelHub/zip_build"
 
 [tool.isort]
 profile = "black"
 known_first_party = "sentinelhub"
 sections = ["FUTURE", "STDLIB", "THIRDPARTY", "LOCALFOLDER"]
 line_length = 120
+skip = ["SentinelHub/resources.py", "SentinelHub/external", "SentinelHub/zip_build"]
```

Comment:
could you perhaps explain these two a bit? Not sure why they are added

Reply:
what is the `resources.py` file? I somehow can't seem to find it, I only see the `resources.qrc`. Is this a dynamically generated file or am I searching in the wrong place?

Reply:
Yes, the resources.qrc is compiled into a resources.py file with pyqt. When you run the deploy you create that file. The external is a folder containing the wheels files, and the zip contains the full plugin to be installed in the Qgis.

Reply:
ah, so the exclusion is not done for the CI, but for the developers, which likely have a local `resources.py` file?